### PR TITLE
Coverage plugin for storm files (SYN-4840)

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: synapse/utils/stormcov/plugin.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        if frame.f_code.co_name != '__init__':
+            return None
+        return self.parent.dynamic_source_filename(None, frame.f_back, force=True)
+
+    def line_number_range(self, frame):
+        return self.parent.line_number_range(frame.f_back)
+
+SHOW_PARSING = False
+
+TOKENS = [
```
Comment (vEpiphyte) [inline synapse/utils/stormcov/plugin.py:118]: Does this map to another token list we have ??? If so, can we reuse it to avoid multi-master problems?

Comment (Cisphyx) [reply synapse/utils/stormcov/plugin.py:118]: Not really unfortunately

File: synapse/lib/ast.py [hunk 2]
```diff
@@ -39,6 +39,7 @@ class AstNode:
     def __init__(self, kids=()):
         self.kids = []
         self.hasast = {}
+        self.lines = (-1, -1)
```
Comment (invisig0th) [inline synapse/lib/ast.py:42]: Could we make this more granular ( ie, the exact text offsets ) and not break the covaregeio use case?

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: synapse/lib/ast.py (hunk 2 above)
Comment (Cisphyx) [reply synapse/lib/ast.py:42]: we could add text offsets for our future use as well, but coverage only cares about lines

File: synapse/tests/test_lib_cell.py [hunk 3]
```diff
@@ -1370,7 +1370,7 @@ async def _iterNewDup(self, user, name=None, remove=False):
 
                     fail = s_t_utils.alist(proxy.iterNewBackupArchive('alreadyrunning', remove=True))
                     await self.asyncraises(s_exc.BackupAlreadyRunning, fail)
-                    await asyncio.wait_for(task, 5)
+                    task.cancel()
```
Comment (vEpiphyte) [inline synapse/tests/test_lib_cell.py:1373]: ?

Comment (Cisphyx) [reply synapse/tests/test_lib_cell.py:1373]: removed, doesn't need to be changed here

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**